.github/workflows/exception-codes.yml

```
@@ -0,0 +1,74 @@
+name: exception-codes
+
+on:
+  schedule:
+    - cron:  '15 */1 * * *'
```

```suggestion
    - cron:  '15 * * * *'
```

I assume, this is enough, no need for `/1` to run monthly.

---

Both variants are the same - execute every hout at minute 15. My variant is more expressiv in this regard. However, will change it to */3 as I guess checking every 3 hours is more than enough.

---

Changed to every 3 hours at minute 15 (which is not really guaranteed anyway by github.